src/main/scala/de/heikoseeberger/sbtfresh/Template.scala

```diff
@@ -128,14 +129,26 @@ private object Template {
       license.fold(fallback)(settings)
     }
 
+    val gitPromptPlugin = if (useGitPrompt) ", GitBranchPrompt" else ""
+    val promptSettings =
+      if (useGitPrompt)
+        ""
+      else
+        """|,
+           |    shellPrompt in ThisBuild := { state =>
+           |      val project = Project.extract(state).currentRef.project
+           |      s"[$project]> "
+           |    }
+        """.stripMargin
+
     s"""|// *****************************************************************************
         |// Projects
         |// *****************************************************************************
         |
         |lazy val $nameIdentifier =
         |  project
         |    .in(file("."))
-        |    .enablePlugins(AutomateHeaderPlugin, GitVersioning)
+        |    .enablePlugins(AutomateHeaderPlugin, GitVersioning$gitPromptPlugin)
```

Comment:
It won't get removed. Depending on the setting a `, GitBranchPrompt` will be appended there. :-)